docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -439,6 +446,12 @@ export const CommentTextBlock = ({
   );
 };
 
+export const getComponentName = (name?: string, children: PropData[] = []) => {
+  if (name && name !== 'default') return name;
+  const ctor = children.filter((child: PropData) => child.name === 'constructor')[0];
+  return ctor?.signatures?.[0]?.type?.name ?? 'default';
+};
```

Comment:
Isn't that addressing this `TODO` in `Video.tsx` file?
https://github.com/expo/expo/pull/16059/files#diff-3a3912aa074e1dccb09a8023e42f918cbf1c802c6fea067a33148d3447165b1aR370-R371

Reply:
Nope, this is required anyway (there were two different problems there - no valid data in JSON & not correctly extracted). If we switch to inline default export the name in JSON file become `new constructor`. 🤷‍♂️